discovery/kubernetes/endpoints.go

```diff
@@ -58,7 +58,7 @@ func NewEndpoints(l log.Logger, svc, eps, pod cache.SharedInformer) *Endpoints {
 	return ep
 }
 
-// Run implements the retrieval.TargetProvider interface.
+// Run implements the discovery.TargetProvider interface.
```

Comment:
The interface is now `discovery.Discoverer` after your refactoring so comments have to be changed accordingly for all SD providers.

Reply:
thanks

Reply:
if you rebase off master, I just fixed this yesterday or the day before :)